bin/deploy-ux-testing.sh

```diff
@@ -30,3 +30,6 @@ cd ..
 # Log into CF and push
 cf login -a $API -u $CF_USER_UX -p $CF_PASSWORD_UX -o $ORG -s $SPACE
 cf push -f manifest-ux-testing.yml
+
+# Migrate and seed the database
+cf run-task hitech-apd-api "npm run migrate && npm run seed"
```

Comment:
We need to come up with a way to tell the UX environment to delete everything from its database on a deployment.  I'd like for it to be something we can do without adding deployed code - that is, I don't want code deployed that will wipe the database.  That seems like it'd be too ripe for abuse (or even an accidental click).

I'm thinking if we install `postgresql-client` as part of this script, we might be able to take advantage of `cf ssh` somehow by mapping the remote database into this script and then connecting and...  blah blah, but that feels less than ideal...

Reply:
how about a command but you have to type out something like DANGEROUSLY_DELETE_DATA for it to run?

Reply:
What if, as part of the staging/production deploy script, we delete the file that deletes everything?  Then it just can't run.  😛  We could have another env var called something like `EMPTY_DATABASE` or something that told the staging seed file to delete stuff.  That way if someone accidentally sets that env var in staging/production, it'll just fail because the file won't be there.